app-frontend/src/app/components/scenes/sceneFilterPane/sceneFilterPane.html

```diff
@@ -14,6 +14,7 @@ <h5>Filters</h5>
   </div>
 </div>
 <div class="sidebar-scrollable">
```

Comment:
If you add a `with-padding` class to this `<div>` then the `filter-pane` class you added won't be needed.

Reply:
Resolved in https://github.com/raster-foundry/raster-foundry/pull/4733/commits/32aea7b0aa17f69614ec7a1feb2cb5480d58f712